docs/source/dev-guide/deep-dives/index.md

```diff
@@ -0,0 +1,15 @@
+# Deep dives
+
+This section contains a series of deep dives into particular gnarly parts
```

Comment:
Particularly gnarly?

Or did you mean "particular" as in "various"?

Reply:
No one knows

Reply:
Here are some other cool words. https://www.mariowiki.com/Special_Zone

I would suggest being boring and replacing gnarly with complex or some other word; perhaps the reader does not agree that the part is gnarly.

Reply:
That's a fair point. We also want to keep in mind Google translate and whether or not "gnarly" would translate well out of English. Likely not. :) Maybe "complex" would be better.

Reply:
Thank you both, this is really appreciated!